### DISCUSSION TITLE
Odd drops to 0 in PID duty cycle %

### DISCUSSION THREAD

**Discussion by hermetheuscoffee** [artisan-roaster-scope/artisan#1071] [General]
Hello, Artisan 2.8.2 user using the built-in software PID, and I'm hoping for input/analysis on the attached roast profile graph. You can see that Artisan PID is setting the duty cycle of the heat to 0% at two different points, where it then recovers gracefully. In looking at the roast data, the BT values all look stable and predictable... the duty cycle simply drops to 0% for some reason. Has anyone else seen this behavior and/or been able to make sense of it?
![image](https://user-images.githubusercontent.com/95500043/211439145-df940505-bbac-42c8-a449-12729994d601.png)

Thanks!
[Brazil_23-01-02_0720.zip](https://github.com/artisan-roaster-scope/artisan/files/10378091/Brazil_23-01-02_0720.zip)

**Comment by poundy** [comment]
with something "glitchy" like this the most important information is what can't be seen in the raw data. 
How long have you been using this roaster on this version of Artisan? Have you seen this before on earlier versions of Artisan? How many roasts have you seen this on, and over what timeframe?

**Reply by orgtech1** [reply]
This roaster has done many roasts on the previous version of Artisan. When using this new version it has happened 3 times

**Reply by orgtech1** [reply]
The previous version used was 2.8.0. The issue didn't show up then but has on 2.8.2

**Comment by MAKOMO** [comment]
Could you please attach the Artisan settings as well?

**Reply by MAKOMO** [reply]
Oh, the Artisan log file could help too

**Reply by MAKOMO** [reply]
..and tell us which version was the "previous" version that worked without this issue. Thanks
